cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -102,27 +109,34 @@ func RelayOracleClaimToEthereum(provider string, contractAddress common.Address,
 
 // initRelayConfig set up Ethereum client, validator's transaction auth, and the target contract's address
 func initRelayConfig(provider string, registry common.Address, event types.Event, key *ecdsa.PrivateKey,
-) (*ethclient.Client, *bind.TransactOpts, common.Address) {
+) (*ethclient.Client, *bind.TransactOpts, common.Address, error) {
 	// Start Ethereum client
 	client, err := ethclient.Dial(provider)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return &ethclient.Client{}, &bind.TransactOpts{}, common.Address{}, err
 	}
 
 	// Load the validator's address
 	sender, err := LoadSender()
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return &ethclient.Client{}, &bind.TransactOpts{}, common.Address{}, err
+
 	}
 
 	nonce, err := client.PendingNonceAt(context.Background(), sender)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
+		return &ethclient.Client{}, &bind.TransactOpts{}, common.Address{}, err
```

Comment:
same here:

I feel like this line should be:
```
return nil, nil, common.Address{}, err
```
due to the function signiture using ponters

Reply:
Accepted.